indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -252,6 +255,37 @@ def _process(self):
                                                     regform=self.regform)
 
 
+class RHRegistrationFormNotificationPreview(RHManageRegFormBase):
+    """Preview registration emails."""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.commit = False
+
+    @no_autoflush
+    def _process(self):
+        message = request.form.get('text', '')
+        state = RegistrationState.get(request.form.get('state', RegistrationState.pending.name))
```

Comment:
likewise, you can nicely get the enum member directly that way

Reply:
Yep, done.